lib/cdo/aws/cloudfront.rb

```diff
@@ -0,0 +1,211 @@
+require_relative '../../../deployment'
```

Comment:
Finished setting up a basic suite of CloudFront integration tests matching the existing Varnish test, please see #4440.